### DISCUSSION TITLE
OnBlur React Native with @react-native-community/datetimepicker

### DISCUSSION THREAD

**Discussion by JanderSilv** [react-hook-form/react-hook-form#1984] [Q&A] [answered]
I'm using the DateTimePicker's package, and I've tried to use it with a Controller for have the OnBlur Validation, but I guess that isn't posible. With the Custom Register in the React Native, we don't have the onBlur and onChange validation yet. There is some way to achieve these kind of validation with the Custom Register?

My solution with the Controller that doesn't work:
```javascript

    const setDate = (args) => {
        setDatePickerIsOpen(false);
        setBirthday(args[1]);
    };

<Controller
      as={
          <DateTimePicker
              mode="date"
              value={new Date()}
              locale="pt-BR"
              onChange={(
                   event,
                   date
              ) =>
                   setDate(
                       event,
                       date
                  )
              }
       />
   }
           control={control}
           name="birthday"
           onChange={(args) =>
                 setDate(args)
           }
           rules={{ required: true }}
           defaultValue={new Date()}
      />
```

**Comment by bluebill1049** [comment, marked as answer]
we are going to support render props with Controller, will that help?

https://github.com/react-hook-form/react-hook-form/releases/tag/v6.0.0-rc.2

**Reply by JanderSilv** [reply]
Definitely, thanks!!
